### PR TITLE
Fix lots registry

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR enhance calculation in the lots registry according stock management : 
- Unique CMM for each lots of an inventory
- Lots with zero MSD are not considered
- Expired lots are not considered

Before : 
image

After : 
image

### CODE REVIEW

File: server/controllers/stock/core.js [hunk 1]
```diff
@@ -489,25 +490,41 @@ function getInventoryQuantityAndConsumption(params) {
...
+    const inventoryLots = _.groupBy(depotInventories, 'inventory_uuid');
+
+    _.map(inventoryLots, (lots) => {
+      // if we don't have the default CMM (avg_consumption) use the
+      // defined or computed CMM for each lots
+      const cmm = _.max(lots.map(lot => lot.avg_consumption));
+
+      // order lots also by ascending quantity
+      // assuming the lot with lowest quantity is consumed first
+      let orderedInventoryLots = _.orderBy(lots, 'quantity', 'asc');
```
Comment (jniles) [inline server/controllers/stock/core.js:506]: These are some nice comments 👍

**Review by jniles — APPROVED** [review]
I'm not sure I understand it all, but the code is clear.  We should work towards writing tests for this registry and/or stock calculations so it is easier to tell what the logic is and what the corner cases are.

bors r+
